protos/mocap/mocap.proto

```diff
@@ -2,78 +2,46 @@ syntax = "proto3";
 
 package mavsdk.rpc.mocap;
 
+import "mavsdk_options.proto";
+
 option java_package = "io.mavsdk.mocap";
 option java_outer_classname = "MocapProto";
 
 /**
- * Motion Capture allow vehicles to navigate when a global
- * position source is unavailable or unreliable
+ * This class allows interfacing a vehicle with a motion capture system in
```

Comment:
```suggestion
 * Allows interfacing a vehicle with a motion capture system in
```

Nitpick: I don't think we need to say "this class", and I'm not sure if that would apply to any kind of bindings.

Reply:
Let me fix it!